cloudinstall/charms/__init__.py

```diff
@@ -332,23 +331,20 @@ class CharmQueue:
     """ charm queue for handling relations in the background
     """
 
-    def __init__(self, ui, config):
+    def __init__(self, ui, config, deployed_charms):
         self.charm_relations_q = Queue()
         self.charm_deploy_q = Queue()
         self.charm_post_proc_q = Queue()
         self.is_running = False
         self.ui = ui
         self.config = config
+        self.deployed_charms = deployed_charms
 
-    def add_relation(self, charm):
-        self.charm_relations_q.put(charm)
-
+    # TODO: Deprecate
```

Comment:
are these TODO comments still needed? AFAICS, you've removed the code they were commenting on